### DISCUSSION TITLE
Upgraded to 2.10.3, and cable connection filtering is broken

### DISCUSSION THREAD

**Discussion by craig-gant** [netbox-community/netbox#5622] [Q&A]
I recently upgraded to 2.10.3, and now when I attempt to connect device interfaces the filtering on the cable connection form isn't working properly. I'll specify Region, Site, Rack, and Device, but I'm still presented with every interface. So for example, I'll take a guess at which Eth47 I'm connecting to and hope it's the correct switch... I don't see the issue reported, and I wanted to check here before I report a bug.

**Comment by ryanmerolle** [comment]
Did this resolve with a more recent version?
